README.md

```diff
@@ -337,7 +337,8 @@ Download the kubeconfig file from your cluster and configure kubectl to use it.
             imagePullPolicy: Always
             name: nginx
           affinity:
-            requiredDuringSchedulingIgnoredDuringExecution:
+            nodeAffinity:
```

Comment:
Done on purpose

Reply:
Sorry, I've skipped the part about broken YAML. Changes removed.